### DISCUSSION TITLE
绳网在一直持续完善吗

### DISCUSSION THREAD

**Discussion by Tu-Doou** [share121/inter-knot#5365] [灌水]
绳网有优化进度和公告说明吗？已经狠狠的期待未来的样子了。
慢一点也没事，主要是真的感兴趣，你们说大伙以后能做管理员吗？
![390b8883a10b481ea3a960ebcdfc190e](https://github.com/user-attachments/assets/9644d668-a524-4e6a-af37-a0650d409aea)

**Comment by Loxia1121** [comment]
有 一直有的 好像在Github还是B站 忘了（）会写下更新了什么&将来可能会有什么新功能之类的

**Reply by ghost** [reply]
会在b站的视频评论区里写因为我就是从b站来的

**Comment by KELERSAMA** [comment]
#4590 这个是更新日志

**Reply by Tu-Doou** [reply]
非常感谢！

**Comment by mephistophelesCLOUD** [comment]
看开发者的B站，这位应该是高一学生，而且大概是独立开发，所以可能没有太多时间和精力吧
